packages/donate-button/src/components/Input/input.css

```diff
@@ -17,7 +17,7 @@
 }
 
 .input--error {
-    border-color: red!important
+    border-color: orangered!important
```

Comment:
Ideally styles wouldn't need !important, I'll add a lint rule banning that later, but it's fine for now given that a lot of the styles will likely be revamped when tackling removing style injector in #77